about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js" defer></script>
+  <title>About us</title>
+</head>
+<body>
+  <!-- Navbar -->
+  <nav class="navbar">
+    <img class="main-logo" id="logo-navbar" src="./images/logo3.svg" alt="Coin Telegraph Web" >
+    <ul class="menu">
+      <li class="list1"><a class="link1" href="./index.html">Home</a></li>
+      <li class="list1"><a class="link1" href="./about.html">About</a></li>
+    </ul>
+    <div class="hamburger">
+      <span class="line"></span>
+      <span class="line"></span>
+      <span class="line"></span>
+    </div>
+  </nav>
```

Comment:
- [ ] As a requirement you should personalize the content by following the design ⚠. There is no top black part of the navigation on your website. Let's add to fix this issue 😉. And also don't forget to hide it on the mobile version.

#### Original design
![image](https://user-images.githubusercontent.com/69011963/152238694-21b35f4c-84d2-43b3-8c1a-a77ae5474129.png)

#### Your website missing part
![image](https://user-images.githubusercontent.com/69011963/152238856-07cd5eb0-d548-4bdf-ac24-5ecadc02ae4a.png)